### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by n0rvik** [FarGroup/FarManager#881]
### Far Manager version

3.0.6389.0

### OS version

10.0.19045.5131

### Steps to reproduce

Edit files in html format.

### Actual behavior

After the reboot, the failure was repeated. 
[bug_report.txt](https://github.com/user-attachments/files/17816287/bug_report.txt)
[far+mdmp.zip](https://github.com/user-attachments/files/17816317/far%2Bmdmp.zip)

**Comment by alabuzhev** [comment]
> Exception: 0xE06D7363 - C++ exception (class std::length_error -> class far_wrapper_exception)
> Details:   string too long -> exception_ptr

"string too long" happens because it is indeed a little too long, about 18 quintillion characters.
18446744073709551615, to be precise.
Which is exactly the same as signed -1, what a coincidence:

![image](https://github.com/user-attachments/assets/6704c6cc-3706-4ebf-a231-fc0e91b614e9)

-1 comes from a plugin esc.dll:

![image](https://github.com/user-attachments/assets/4244ca9c-324f-4ee7-8066-73e4f289c345)

Because esc.dll apparently tries to kill spaces in the string:

![image](https://github.com/user-attachments/assets/9dac859b-588d-4bb2-9735-4f429b96148d)

Perhaps it tries a little too eagerly.

Figuring out what exactly is wrong with that line and why it's doomed to fail spectacularly is left as an exercise for the reader.

TL;DR: remove esc.dll, report it [here](https://github.com/trexinc/evil-programmers/).
